### PR TITLE
Improve citation section of Readme

### PR DESCRIPTION
**Pull request by orzechow**
This collapses all but one citation for better appearance.
It also adds my thesis to the citations.

### CODE REVIEW

**Comment by orzechow** [comment]
@ll-nick

File: README.md [hunk 1]
```diff
@@ -316,7 +318,39 @@ If you use arbitration graphs in your research, we would be pleased if you cite
...
+
+<details>
+<summary>Earlier publications</summary>
+
+A safety concept that extends Arbitration Graphs with behavior verification and fallback layers in the context of automated driving has been proposed by Piotr Spieker (née Orzechowski) in his PhD thesis.
+This served as the basis for the paper with Nick above.
+
+_Piotr F. Orzechowski, “Verhaltensentscheidung für automatisierte Fahrzeuge mittels Arbitrationsgraphen,” phd, Karlsruher Institut für Technologie (KIT), 2023. doi: [10.5445/IR/1000160638](https://doi.org/10.5445/IR/1000160638)._
+
+{% raw %}
```
Comment (ll-nick) [inline README.md:333]: Just like in #83, these don't look great on the GitHub rendered page.

**Comment by ll-nick** [comment]
I just wanted to view these changes on locally built pages but I get the following error when starting the container. Any idea if that has something to do with this PR? Last week, this still worked.

```
Attaching to github-pages-dev-1
github-pages-dev-1  | bundler: failed to load command: jekyll (/usr/local/bundle/bin/jekyll)
github-pages-dev-1  | /usr/local/bundle/gems/bundler-2.5.21/lib/bundler/definition.rb:599:in `materialize': Could not find rexml-3.3.8, faraday-2.12.0, execjs-2.9.1, activesupport-7.2.1, faraday-net_http-3.3.0, json-2.7.2, uri-0.13.1 in locally installed gems (Bundler::GemNotFound)
github-pages-dev-1  |   from /usr/local/bundle/gems/bundler-2.5.21/lib/bundler/definition.rb:193:in `specs'
github-pages-dev-1  |   from /usr/local/bundle/gems/bundler-2.5.21/lib/bundler/definition.rb:260:in `specs_for'
github-pages-dev-1  |   from /usr/local/bundle/gems/bundler-2.5.21/lib/bundler/runtime.rb:18:in `setup'
github-pages-dev-1  |   from /usr/local/bundle/gems/bundler-2.5.21/lib/bundler.rb:164:in `setup'
github-pages-dev-1  |   from /usr/local/bundle/gems/bundler-2.5.21/lib/bundler/setup.rb:32:in `block in <top (required)>'
github-pages-dev-1  |   from /usr/local/bundle/gems/bundler-2.5.21/lib/bundler/ui/shell.rb:173:in `with_level'
github-pages-dev-1  |   from /usr/local/bundle/gems/bundler-2.5.21/lib/bundler/ui/shell.rb:119:in `silence'
github-pages-dev-1  |   from /usr/local/bundle/gems/bundler-2.5.21/lib/bundler/setup.rb:32:in `<top (required)>'
github-pages-dev-1  |   from /usr/local/bundle/gems/bundler-2.5.21/lib/bundler/cli/exec.rb:56:in `require_relative'
github-pages-dev-1  |   from /usr/local/bundle/gems/bundler-2.5.21/lib/bundler/cli/exec.rb:56:in `kernel_load'
github-pages-dev-1  |   from /usr/local/bundle/gems/bundler-2.5.21/lib/bundler/cli/exec.rb:23:in `run'
github-pages-dev-1  |   from /usr/local/bundle/gems/bundler-2.5.21/lib/bundler/cli.rb:455:in `exec'
github-pages-dev-1  |   from /usr/local/bundle/gems/bundler-2.5.21/lib/bundler/vendor/thor/lib/thor/command.rb:28:in `run'
github-pages-dev-1  |   from /usr/local/bundle/gems/bundler-2.5.21/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
github-pages-dev-1  |   from /usr/local/bundle/gems/bundler-2.5.21/lib/bundler/vendor/thor/lib/thor.rb:527:in `dispatch'
github-pages-dev-1  |   from /usr/local/bundle/gems/bundler-2.5.21/lib/bundler/cli.rb:35:in `dispatch'
github-pages-dev-1  |   from /usr/local/bundle/gems/bundler-2.5.21/lib/bundler/vendor/thor/lib/thor/base.rb:584:in `start'
github-pages-dev-1  |   from /usr/local/bundle/gems/bundler-2.5.21/lib/bundler/cli.rb:29:in `start'
github-pages-dev-1  |   from /usr/local/bundle/gems/bundler-2.5.21/exe/bundle:28:in `block in <top (required)>'
github-pages-dev-1  |   from /usr/local/bundle/gems/bundler-2.5.21/lib/bundler/friendly_errors.rb:117:in `with_friendly_errors'
github-pages-dev-1  |   from /usr/local/bundle/gems/bundler-2.5.21/exe/bundle:20:in `<top (required)>'
github-pages-dev-1  |   from /usr/local/bundle/bin/bundle:25:in `load'
github-pages-dev-1  |   from /usr/local/bundle/bin/bundle:25:in `<main>'
github-pages-dev-1 exited with code 1
```

**Comment by orzechow** [comment]
Just for the record (we solved this via chat):
These Jekyll issues arise from Gem caches incompatible with upstream changes of Gem dependencies. 

Simply `git clean` your repo and restart the local Jekyll server.

File: README.md [hunk 2]
```diff
@@ -316,7 +318,39 @@ If you use arbitration graphs in your research, we would be pleased if you cite
   doi = {10.48550/arXiv.2411.10170},
   url={https://arxiv.org/abs/2411.10170}, 
 }
+```
+{% endraw %}
+
+
+<details>
+<summary>Earlier publications</summary>
```
Comment (ll-nick) [inline README.md:324]: The entire section does not render properly for me via Jekyll. Looks fine on the pure markdown rendering and generally I think it's a good idea to use spoilers here.

File: README.md [hunk 3]
```diff
@@ -316,7 +318,39 @@ If you use arbitration graphs in your research, we would be pleased if you cite
...
+
+A safety concept that extends Arbitration Graphs with behavior verification and fallback layers in the context of automated driving has been proposed by Piotr Spieker (née Orzechowski) in his PhD thesis.
```
Comment (ll-nick) [inline README.md:326]: I'd add horizontal dividers between the publications to distinguish more easily which part belongs where. See also the analogous suggestions below.

```suggestion
---
A safety concept that extends Arbitration Graphs with behavior verification and fallback layers in the context of automated driving has been proposed by Piotr Spieker (née Orzechowski) in his PhD thesis.
```

File: README.md [hunk 4]
```diff
@@ -328,7 +362,15 @@ If you use arbitration graphs in your research, we would be pleased if you cite
   issn = {2642-7214},
   doi = {10.1109/IV47402.2020.9304723},
 }
+```
+{% endraw %}
 
+The foundations for Arbitration Graphs have been proposed in the context of robot soccer:
```
Comment (ll-nick) [inline README.md:362]:
```suggestion
---
The foundations for Arbitration Graphs have been proposed in the context of robot soccer:
```

**Comment by ll-nick** [comment]
Let me try again tomorrow. I think I did but let me test it in a clean state to make sure the fault is not in my local setup.

**Comment by ll-nick** [comment]
Just for reference:
> Simply git clean your repo and restart the local Jekyll server.
This did not actually work. I had to `rm Gemfile.lock` to fix it.

**Comment by ll-nick** [comment]
Ok,

**Comment by orzechow** [comment]
Follow-up PR: #86
